src/components/pages/Restaurant/FoodModal.tsx

```diff
@@ -77,6 +78,15 @@ function FoodModal({ foodId, food_quantity, toggled }: IModalProps) {
       if (foodQuantity !== 0) {
         const { restaurant_slug } = router.query;
         dispatch(addFoodToCartRequest(food, restaurant_slug, foodQuantity));
+        toast.error(`${food.title} foi adicionado na sacola!`, {
```

Comment:
I just think it would be more interesting if we use 'toast.success()' instead of 'toast.error()', because the green color better represents the idea of successful action.

But feel free to accept or not this suggestion 😁

Reply:
Hi dear, i was going to put the success but with ifood is red i put error haha. Im going to change that right now, green color it's so more better than red on this case!! Tks broo

Reply:
Great!!

Thanks for the contribution, bro!! 🚀

I'm going to follow you right now haha